### DISCUSSION TITLE
read table columns with format bytea (where .jpg data is stored) and use it as TexureImage-Data

### DISCUSSION THREAD

**Discussion by felix7700** [Marzin-bot/PostgreSQLClient#43] [General]
Hello again,

Now I have a new PSQL problem, I don't know if there is a solution to this problem, but I want this to work:
Read table-columns with Bytea format (where .jpg data is stored) and use it as TexureImage data

Thanks for your help!

**Comment by KuwazyStudio** [comment]
Hello, you have a "load_jpg_from_buffer()" method in the Image class. I believe this is what you are looking for. https://docs.godotengine.org/fr/stable/classes/class_image.html#class-image-method-load-jpg-from-buffer

**Comment by KuwazyStudio** [comment]
I believe that I will create a tutorial on this case later because it is interesting.
